src/ClassicUO.Client/Game/Managers/MacroManager.cs

```diff
@@ -736,6 +736,19 @@ private int Process(MacroObject macro)
                                     break;
 
                                 case MacroSubType.Journal:
+                                    if(ProfileManager.CurrentProfile.UseAlternateJournal)
+                                    {
+                                        ResizableJournal rjournal = UIManager.GetGump<ResizableJournal>();
+                                        if (macro.Code == MacroType.Close)
+                                        {
+                                            rjournal?.Dispose();
+                                        }
+                                        else if (macro.Code == MacroType.Open)
```

Comment:
The `MacroType.Open` should be handled by the `GameActions.OpenJournal` function like for std journal:
https://github.com/ClassicUO/ClassicUO/blob/a244efff3822c9186918b70cc83a289a21cd242c/src/ClassicUO.Client/Game/Managers/MacroManager.cs#L518